### PR TITLE
Consolidating Roles

### PR DESCRIPTION
**Pull request by mew2057**
Implemented  [MDD] Role Consolidation (#98).

- Bumped Version to 1.0.1
- Moved non operator  roles into image.

### CODE REVIEW

**Comment by whowutwut** [comment]
If you're bumping up versions, there's a hard code in the `conf.py` of the docs we should probably  keep lock step so we can `git grep` and find the versions to modify. It's a 0.9.2 i think...

**Comment by mew2057** [comment]
I have an ansible playbook for handling version bumps: `ansible/versioning-playbook.yaml`. It looks like I just left the  `conf.py` out.

**Comment by mew2057** [comment]
Should also  note this PR handles some of  the cv linter errors:

```
------------------------------------------------------------------------------------------------------------
==> Linting results for operator ibm-spectrum-scale-csi-operator
[REVIEW] deploy/operator.yaml: spec.template.spec.containers[0].readinessProbe not defined (ContainerWithNoMatchingServiceHasReadinessProbe)
[REVIEW] deploy/operator.yaml: spec.template.spec.containers[1].readinessProbe not defined (ContainerWithNoMatchingServiceHasReadinessProbe)
[WARNING] deploy/service_account.yaml: no imagePullSecrets defined, pods will not be able to pull namespace-scoped images from the local registry (ServiceAccountHasPullSecret)
[ERROR] deploy/crds: example custom resource not found (OperatorHasRequiredResources)
[ERROR] deploy/crds/ibm-spectrum-scale-csi-operator-crd.yaml, deploy/olm-catalog/ibm-spectrum-scale-csi-operator/1.0.0/ibm-spectrum-scale-csi-operator-crd.yaml, deploy/olm-catalog/ibm-spectrum-scale-csi-operator/1.0.1/ibm-spectrum-scale-csi-operator-crd.yaml: resource metadata.name "csiscaleoperators.csi.ibm.com" not unique among CustomResourceDefinition resources (ResourceHasUniqueNameAmongKind)
[ERROR] deploy/namespace.yaml: ["app.kubernetes.io/instance" "app.kubernetes.io/managed-by" "app.kubernetes.io/name"] not defined under metadata.labels (RequiredMetadataLabelsDefined)
[ERROR] deploy/olm-catalog/csv-config.yaml: invalid schema for kube version 1.11.0: [apiVersion not set, kind not set] (ResourceSchemaIsValid)
[ERROR] deploy/olm-catalog/csv-config.yaml: invalid schema for kube version 1.12.4: [apiVersion not set, kind not set] (ResourceSchemaIsValid)
[ERROR] deploy/olm-catalog/csv-config.yaml: spec.validation not defined (CustomResourceIsValid)
[ERROR] deploy/operator.yaml: neither spec.template.spec.containers[0].resources.limits.cpu nor spec.template.spec.containers[0].resources.requests.cpu is defined (ContainerDefinesResources)
[ERROR] deploy/operator.yaml: neither spec.template.spec.containers[1].resources.limits.cpu nor spec.template.spec.containers[1].resources.requests.cpu is defined (ContainerDefinesResources)
[ERROR] deploy/operator.yaml: spec.template.spec.containers[0].livenessProbe not defined (ContainerHasLivenessProbe)
[ERROR] deploy/operator.yaml: spec.template.spec.containers[0].resources.limits.memory not defined (ContainerDefinesResources)
[ERROR] deploy/operator.yaml: spec.template.spec.containers[0].resources.requests.memory not defined (ContainerDefinesResources)
[ERROR] deploy/operator.yaml: spec.template.spec.containers[1].livenessProbe not defined (ContainerHasLivenessProbe)
[ERROR] deploy/operator.yaml: spec.template.spec.containers[1].resources.limits.memory not defined (ContainerDefinesResources)
[ERROR] deploy/operator.yaml: spec.template.spec.containers[1].resources.requests.memory not defined (ContainerDefinesResources)

------------------------------------------------------------------------------------------------------------
==> Lint Summary

Rule                                             Severity  Total   Reduced  Ignored
ContainerDefinesResources                        ERROR     6       0        0
ContainerHasLivenessProbe                        ERROR     2       0        0
ResourceSchemaIsValid                            ERROR     2       0        0
CustomResourceIsValid                            ERROR     1       0        0
OperatorHasRequiredResources                     ERROR     1       0        0
RequiredMetadataLabelsDefined                    ERROR     1       0        0
ResourceHasUniqueNameAmongKind                   ERROR     1       0        0
ServiceAccountHasPullSecret                      WARNING   1       0        0
ContainerWithNoMatchingServiceHasReadinessProbe  REVIEW    2       0        0
```

File: stable/ibm-spectrum-scale-csi-operator-bundle/operators/ibm-spectrum-scale-csi-operator/hacks/package_operator.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 PACKAGE_POSTFIX="package.yaml"
 
-#1.0.0
+#1.0.1
```
Comment (whowutwut) [inline stable/ibm-spectrum-scale-csi-operator-bundle/operators/ibm-spectrum-scale-csi-operator/hacks/package_operator.py]: If this is a comment, just remove it completely?

Comment (mew2057) [reply stable/ibm-spectrum-scale-csi-operator-bundle/operators/ibm-spectrum-scale-csi-operator/hacks/package_operator.py]: yeah, we probably should

**Pull request merged**